www/source/docs/index.html.slim

```diff
@@ -10,113 +10,63 @@ section.docs--hero
-          ul.no-bullet
-            li
-              a href="https://habitat.sh/docs"
-                | Documentation
-        .body-content
-          h4
-            a href="/docs/reference#handlebars-helpers"
-              | Lifecycle Hook Helpers
-          p
-            | All of the QoL Handlebars helpers for use in your hooks
-            | in one convenient place.
-
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="https://habitat.sh/blog"
-                | Blog
-        .body-content
-          h4
-            a href="/blog/2017/09/a-journey-with-habitat-on-amazon-ecs-part1"
-              | Hab + ECS
-          p
-            | Want to learn how to use habitat with Amazon's cloud-based
-            | scheduler? Look no further , we've got you covered.
+      .docs--get-started--content
+        h2 Getting Started
 
-  .row.docs--quicklink--cards
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="/docs/best-practices"
-                | Guide
-        .body-content
-          h4
-            a href="/docs/developing-packages"
-              | Plan Guide
-          p
-            | Get started with how to write Habitat plans
-            | here.
+  section.get-started--boxes
+      div.row
+        div.columns.medium-12
+          a.get-started--boxes--item.card href="/docs/reference/#infographics"
+            div.columns.medium-6
+              h3.h3 Habitat in Pictures
+              p
+                | Take a high-level look at the entire Habitat landscape and workflow using this set of diagrams
+            div.columns.medium-6
+              img src="/images/infographics/habitat-diagrams-thumb.png" alt="Thumbnail of Habitat system diagrams."
 
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="https://habitat.sh/blog"
-                | Blog
-        .body-content
-          h4
-            a href="https://www.habitat.sh/blog/2017/07/Habitat-mysql-cluster/"
-              | Stateful Services with Habitat
-          p
-            | Habitat provides the same cloud native experience for
-            | both containerized and stateful services. Find out how.
+      div.row
+        div.columns.medium-12
+          a.get-started--boxes--item.card href="/learn/"
+            div.columns.medium-6
+              h3.h3 Demos and Tutorials
+              p
+                | These short demos will first walk you through installing Habitat then learning how your team build, deploy, and manage all of your applications - both new and legacy - in a cloud-native way
```

Comment:
I'm stumbling over this language a bit. How about `These short demos will walk you through installing Habitat, then show you how to build, deploy and manage all of your applications...`